templates/azure/index.html

```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
+  </section>
+
+  <section class="p-section">
     <div class="row--50-50">
       <hr class="p-rule--muted" />
+      <div class="col">
+        <h2>Secure Data at Run-Time with Confidential Computing on Azure</h2>
+      </div>
+      <div class="col">
+        <p>If you're running critical workloads and need to process sensitive data, security at system memory is essential. With confidential computing, your workloads are  encrypted at run time, and are thus secure from all software vulnerabilities outside the VM perimeter - including the hypervisor and from unauthorized cloud admins. Pair confidential VMs with Ubuntu Pro for security in and out of your VM.</p>
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Secure during compute</li>
+          <li class="p-list__item is-ticked">Enables zero trust</li>
+          <li class="p-list__item is-ticked">Process sensitive workloads without worry</li>
+          <li class="p-list__item is-ticked">Available on Ubuntu Pro for added security or the free versions of Ubuntu</li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p><a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-confidential-vm-jammy?tab=Overview" class="p-button">Launch an Ubuntu Pro CVM now</a></p>
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts?tab=Overview" style="margin-right: 1rem">Launch an Ubuntu CVM&nbsp;&rsaquo;</a>
+        <a href="/confidential-computing">Learn more about CVMs&nbsp;&rsquo;</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--25-75">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2 class="p-muted-heading"><a href="/blog/tag/ubuntu-on-azure">Latest from our blog&nbsp;&rsaquo;</a></h2>
+      </div>
+      <div class="col">
+        <div id="blog-posts" class="row"></div>
+      </div>
+
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <hr class="p-rule" />
       <div class="col">
         <h2>Optimise your cloud journey with Canonical</h2>
       </div>
       <div class="col">
-        <p>
-          Canonical works with the top enterprises and innovators in the world to define and execute their cloud journeys. Migration, application redesign, support and optimisation services are some of the ways in which we have helped customers make the most of Ubuntu and Azure together.
-        </p>
+        <p>Canonical works with the top enterprises and innovators in the world to define and execute their cloud journeys. Migration, application redesign, support and optimization services are some of the ways in which we have helped customers make the most of Ubuntu and Azure together.</p>
         <hr class="p-rule--muted" />
         <a href="/azure/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on Azure</a>
```

Comment:
You could use the [Vanilla cta block](https://vanillaframework.io/docs/patterns/cta-block) here

Reply:
Done